api/envoy/extensions/http/header_formatters/preserve_case/v3/preserve_case.proto

```diff
@@ -17,7 +17,19 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // See the :ref:`header casing <config_http_conn_man_header_casing>` configuration guide for more
 // information.
 message PreserveCaseFormatterConfig {
+  enum FormatterTypeOnUnknownHeaders {
+    // Use LowerCase on unknown headers.
+    DEFAULT = 0;
+
+    // Use ProperCaseHeaderKeyFormatter on unknown headers.
```

Comment:
Can you describe the behavior more in the API docs like "upper case the first character in each word" on `PROPER_CASE`?

Reply:
done